app/controllers/games_controller.rb

```diff
@@ -36,6 +36,13 @@ def update
     end
   end
 
+  def forfeit
+    @game = Game.find_by(id: params[:id])
+    @game.forfeit(current_user)
+    flash[:alert] = 'You have forfeited the game.'
+    redirect_to games_path
```

Comment:
I'm thinking we can add a modal in the game_show_page to create a pop-up message that verify the user if they want to forfeit. If they choose to forfeit again, then we do `redirect_to games_path, notice: "You have forfeited the game`.